md-dao/src/main/java/com/wipro/ats/bdre/md/dao/LineageQueryDAO.java

```diff
@@ -49,6 +52,45 @@
         return lineageQuerys;
     }
 
+    //get Instance exec ids for the process id from LQ table
+    private Long getInstanceExecIds(Integer processId) {
+        Long instanceExecId = null;
+        Session session = sessionFactory.openSession();
+        session.beginTransaction();
+        Criteria getLastElementCriteria = session.createCriteria(LineageQuery.class).add(Restrictions.eq("processId", processId)).addOrder(Order.desc("instanceExecId"));
+
+        if(getLastElementCriteria.list().size() != 0) {
+            LineageQuery lineageQuery = (LineageQuery) getLastElementCriteria.list().get(0);
+            instanceExecId = lineageQuery.getInstanceExecId();
+        }
+        session.getTransaction().commit();
+        session.close();
+        return instanceExecId;
+    }
+
+    public List<LineageQuery> getLastInstanceExecLists(Integer processId) {
+        Long instanceExecId = null;
+        List<LineageQuery> lineageQueryList = new ArrayList<LineageQuery>();
+        //int counter = 1;
+        System.out.println("out");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS106)